flink-table/flink-table-planner-blink/src/main/java/org/apache/flink/table/planner/operations/SqlToOperationConverter.java

```diff
@@ -244,6 +224,117 @@ private Operation convertCreateTable(SqlCreateTable sqlCreateTable) {
 			sqlCreateTable.isTemporary());
 	}
 
+	private CatalogTable createCatalogTable(SqlCreateTable sqlCreateTable) {
```

Comment:
Can we stop letting this file grow infinitely and somehow move those static methods into separate classes?